### ISSUE TITLE
Parser bug? Narrator not being pulled from 511?

### ISSUE THREAD

**Issue by Arty-chan** [BCLibCoop/nnels#111]
So got the sys control numbers parsed and then did the s3 path batch update, but when generating the files, I noticed that there was no narrator on any of the files.

Example:
https://nnels.ca/items/winston-churchill-reporting-adventures-young-war-correspondent

the information is in the MARCXML
> marc:datafield tag="511" ind1="0" ind2=" " marc:subfield code="a" Read by Simon Vance.

Full set: https://nnels.ca/admin/content/cals/records-sets/php-2016-04-full-catalogue

**Comment by davekisly** [comment]
hi @Arty-chan - really? ugh, yikes, will review tomorrow.

**Comment by davekisly** [comment]
Ok, that was a small regression error. Its now fixed & committed to both DEV and PROD. I've tested here: https://dev.nnels.ca/items/harperland-politics-control

**Comment by Arty-chan** [comment]
thanks. Don't need to fix today (though this weekend would be great if you have time to take a look).

I noticed that when I tested on dev, it stripped the period on the narrator's middle initial.

Original text:
> Read by David M. Adams.
turns into:
> David M Adams
see: https://dev.nnels.ca/items/gandhis-path-bob-swanns-work-peace-and-community-e

I want the parser to stripe trailing periods, but not all periods.
